docs/source/examplecleanup.rst

```diff
@@ -0,0 +1,116 @@
+Would you like to know more?
+============================
+
+End-to-End Example
+------------------
+
+An End-to-End example walkthrough of tartufo cleanup
```

Comment:
Could you expand this a little bit? Just saying `tartufo cleanup` I think begs the question, "How does this differ from `tartufo --cleanup`?"

Maybe saying something like, "Now that you have found secrets in your repo, what do you do with that information?" ...but, you know, maybe a bit better wording than that. :smile: